plexapi/video.py

```diff
@@ -557,7 +557,7 @@ def season(self, title=None, season=None):
 
     def seasons(self, **kwargs):
         """ Returns a list of :class:`~plexapi.video.Season` objects in the show. """
-        key = f'{self.key}/children?excludeAllLeaves=1'
+        key = f'{self.key}/children?excludeAllLeaves=1&X-Plex-Container-Size={self.childCount}'
         return self.fetchItems(key, Season, **kwargs)
```

Comment:
Add `container_size` argument to `fetchItems()` instead of hard-coding it into the key.

Reply:
Changed.